<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageAddStakingAccount</class>
 <widget class="QWidget" name="PageAddStakingAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #FFFFFF;
color:black;</string>
  </property>
  <widget class="QFrame" name="headerFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #FCD535;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #474D57;</string>
    </property>
    <property name="text">
     <string>Add new staking account</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="backPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: solid;
	background-color: rgba(255, 255, 255, 16);
	border-radius: 5px;
	border-width: 1px;
	border-color: rgb(235,235,235);
}
QPushButton:hover {
    background-color: rgba(255, 255, 255, 64);
}

QPushButton:pressed {
    background-color: rgba(255, 255, 255, 128);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../../res.qrc">
      <normaloff>:/res/ic/res/ic/arowLeft.png</normaloff>:/res/ic/res/ic/arowLeft.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="stakeForGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	border-style: solid;
	background-color: transparent;
	border-radius: 5px;
	border-width: 1px;
	border-color: lightgrey;
	color: darkgray;
}</string>
   </property>
   <property name="title">
    <string>Select to stake for</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>231</width>
      <height>35</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-radius: 5px;
border-width: 1px;
border-color: lightgrey;
background-color: rgb(255, 254, 173);</string>
    </property>
    <property name="text">
     <string>STATUS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="amountGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	border-style: solid;
	background-color: transparent;
	border-radius: 5px;
	border-width: 1px;
	border-color: lightgrey;
	color: darkgray;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="maxPushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-style: solid;
	background-color: rgba(0, 0, 0, 8);
	border-radius: 5px;
	border-width: 1px;
	border-color: lightgrey;
}
QPushButton:hover {
    background-color: rgba(0, 0, 0, 16);
}

QPushButton:pressed {
    background-color: rgba(0, 0, 0, 64);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../../res.qrc">
      <normaloff>:/res/ic/res/ic/max.png</normaloff>:/res/ic/res/ic/max.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="amountLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit[text=&quot;&quot;]{ 
	color:gray;
 }
QLineEdit{ 
	color:black; 
	border-style: solid;
	border-radius: 5px;
	border-width: 1px;
	border-color: lightgrey;
    padding-left: 4px;
    padding-right: 4px;	
}
</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly|Qt::ImhNoPredictiveText|Qt::ImhPreferNumbers</set>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Amount</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="availablePushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color:green;
border-color: transparent;
text-align:right;
padding-top:0px;
padding-bottom:0px;
</string>
    </property>
    <property name="text">
     <string>Available: 0 LYR</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="nameDaysGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	border-style: solid;
	background-color: transparent;
	border-radius: 5px;
	border-width: 1px;
	border-color: lightgrey;
	color: darkgray;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit[text=&quot;&quot;]{ 
	color:gray;
 }
QLineEdit{ 
	color:black; 
	border-style: solid;
	border-radius: 5px;
	border-width: 1px;
	border-color: lightgrey;
    padding-left: 4px;
    padding-right: 4px;	
}
</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhLatinOnly|Qt::ImhSensitiveData</set>
    </property>
    <property name="maxLength">
     <number>32</number>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Name of staking account</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="daysLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit[text=&quot;&quot;]{ 
	color:gray;
 }
QLineEdit{ 
	color:black; 
	border-style: solid;
	border-radius: 5px;
	border-width: 1px;
	border-color: lightgrey;
    padding-left: 4px;
    padding-right: 4px;	
}
</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly|Qt::ImhNoPredictiveText|Qt::ImhPreferNumbers</set>
    </property>
    <property name="maxLength">
     <number>6</number>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Days to stake from 3 to 365000 days</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="previewPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>291</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: solid;
	background-color: rgba(252, 213, 53, 255);
	border-radius: 5px;
	border-width: 1px;
	border-color: rgb(235, 235, 235);
	color: black;
}
QPushButton:hover {
    background-color: rgba(252, 213, 53, 192);
}

QPushButton:pressed {
    background-color: rgba(252, 213, 53, 128);
}

QPushButton:disabled {
    color: rgba(0, 0, 0, 128);
}</string>
   </property>
   <property name="text">
    <string>PREVIEW</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../../res.qrc"/>
 </resources>
 <connections/>
</ui>
